test/penalizer/Utils.ts

```diff
@@ -0,0 +1,170 @@
+import { ether } from '@openzeppelin/test-helpers'
+import { Transaction } from 'ethereumjs-tx'
+import { ethers } from 'ethers'
+import { toChecksumAddress } from 'web3-utils'
+import { RelayRequest } from '../../src/common/EIP712/RelayRequest'
+import TypedRequestData, { getDomainSeparatorHash } from '../../src/common/EIP712/TypedRequestData'
+import { Commitment } from '../../src/enveloping/Commitment'
+import { AccountKeypair } from '../../src/relayclient/AccountManager'
+import { Address } from '../../src/relayclient/types/Aliases'
+import { RelayHubInstance, TestVerifierEverythingAcceptedInstance, TestTokenInstance, IForwarderInstance } from '../../types/truffle-contracts'
+import { getLocalEip712Signature } from '../../Utils'
+
+const gasPrice = '1'
+
+interface RelayRequestParams{
+  from: Address
+  to: Address
+  relayData: string
+  enableQos?: boolean
+}
+
+interface CommitmentReceipt {
```

Comment:
this isn't the same `CommitmentReceipt` as the one in `enveloping/Commitment.ts`. if it was, we wouldn't be able to use it as a parameter for the `claim` calls from the javascript side. 

the reason for this is that the other `CommitmentReceipt` includes a `Commitment`-typed variable, which is a class (and not an interface). the `claim` function—as with other bindings—expects plain objects, and commitments do not meet this criterion due to having behavior (i.e. the `encodedFields` & `encodeForSign` functions).

as a result, this interface is defined exactly as in `types/truffle-contracts/index.d.ts:4194`.

if anyone knows of a way to import this instead of duplicating it, please lmk